RFR-App/RFR/InitializationView.swift

```diff
@@ -106,9 +106,8 @@ let liveViewModel = LiveViewModel(
 )
 
 let voucherViewModel2 = VoucherViewModel(
-    authenticator: authenticator,
-    url: incentivesUrl,
-    dataStoreStack: mockDataStoreStack
+    vouchers: MockVouchers(count: 3, voucher: Voucher(code: "test-voucher")),
```

Comment:
I guess because this is below `#if DEBUG`, this mock data is not in the release app?

Reply:
yes